jwst/pipeline/resample.cfg

```diff
@@ -7,4 +7,5 @@ pixfrac = 1.0
 kernel = 'square'
 fillval = 'INDEF'
 good_bits = 4
-blendheaders = True
\ No newline at end of file
+blendheaders = True
+suffix = 'resamp'
```

Comment:
Resampled products should use a suffix of 'i2d' or 's2d', depending on whether they're images or spectra.

Reply:
Ah, right.  this suggests that I should have a separate `resample_spec.cfg`, and then have specific suffixes appropriate to each.